src/main/scala/org/dbpedia/spotlight/wikistats/RawWikiMain.scala

```diff
@@ -50,7 +50,7 @@ object RawWikiMain {
 
     val rawWikiStats = new RawWikiStats(lang)
 
-    rawWikiStats.buildRawWiki(wikipediaParser)
+    rawWikiStats.buildRawWiki(wikipediaParser).collect().foreach(println)
```

Comment:
does the `println` do any work here besides debuging ? If not it might be better to put logging statements instead ?